src/js/control-bar/text-track-controls/descriptions-button.js

```diff
@@ -0,0 +1,25 @@
+import CaptionsButton from './captions-button.js';
+
+/**
+ * The button component for toggling and selecting descriptions
+ *
+ * @constructor
+ */
+class DescriptionsButton extends CaptionsButton {
```

Comment:
As @gkatsev says, subclassing CaptionsButton is to get the "descriptions settings" menu item. I guess it could be refactored into a TextTrackWithSettingsButton, which is a subclass of TextTrackButton, and of which CaptionsButton and DescriptionsButton are a subclass?

In this implementation, Captions and descriptions share the same style options. This was a compromise between not styling the descriptions at all (since the main use case for descriptions is to render them in a non-visual way with a screen reader), and adding the complexity of a whole separate set of styling options.

The default of having descriptions rendered visibly is mostly so that people without a screen reader can get an idea what descriptions are, and don't get confused by a control button which would seem to do nothing if the descriptions were rendered invisibly by default.

Since descriptions are similar to captions, it makes sense to style them as captions. A more sophisticated option would style them differently from captions and allow visual presentation of both descriptions and captions at the same time, but watching both at once isn't a realistic use case; the amount of work to implement that option doesn't make sense.